packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -149,6 +149,24 @@ class FlutterPlugin implements Plugin<Project> {
             }
         }
 
+        if (project.hasProperty("multi-dex-enabled") &&
+            project.property("multi-dex-enabled").toBoolean() &&
+            project.android.defaultConfig.minSdkVersion <= 20) {
+            project.android {
+                defaultConfig {
+                    multiDexEnabled true
+                }
+                buildTypes {
+                    release {
+                        multiDexKeepFile project.file('multidex-flutter-keepfile.txt')
```

Comment:
could this file be moved to the flutter installation? 

flutter_proguard_rules.pro had a similar issue:
https://github.com/flutter/flutter/blob/e7a9adeeabccf99d67321dc7a87fdf3820f06912/packages/flutter_tools/gradle/flutter.gradle#L188

Reply:
Yeah!